app/views/gene.html

```diff
@@ -150,12 +150,12 @@ <h3>Background:
             <div class="panelContent" id="{{mutation.mutation_effect_uuid}}">
               <p ng-show="displayPrecisely(mutation.mutation_effect.oncogenic_uuid)">
                 <strong style="float: left; margin-right: 5px;">Oncogenic:</strong>
-                <realtime-string t="'checkbox'" key="'oncogenic'" path="effectPath" checkboxes="checkboxes['oncogenic']" uuid="getUUID({type: 'mutation_effect', key: 'oncogenic', path: path})"></realtime-string>
```

Comment:
For radio, we can never deselect the options anymore. I don't think that's our intension. For instance, you may not want to select any oncogenicity for `Oncogenic Mutations`. But if the curator accidentally chooses one, there is no way to revert back.

Reply:
We can revert it back but I have to do that programmatically. I use radio because it fits the case that only choose at most 1 checkbox in a checkbox group.